src/ceph.in

```diff
@@ -259,6 +259,9 @@ def parse_cmdargs(args=None, target=''):
 
     parser.add_argument('-c', '--conf', dest='cephconf',
                         help='ceph configuration file')
+
+    parser.add_argument('-k', '--keyring', dest='keyring',
```

Comment:
@Liuchang0812 if one host need connect more than one ceph cluster should specify ceph.conf and ceph.keyring file